pkg/workflows/wasm/host/module.go

```
@@ -470,6 +478,10 @@ func (m *module) Execute(ctx context.Context, request *wasmpb.ExecuteRequest) (*
 	return nil, errors.New("not implemented")
 }
 
+func (m *module) SetCapabilityExecutor(handler CapabilityExecutor) {
```

What consumer benefits from this API? Consider an error return val?

`func (m *module) SetCapabilityExecutor(handler CapabilityExecutor) error {`

---

The Engine calls it. Sure, we can return an error.